front/components/vaults/VaultResourcesList.tsx

```diff
@@ -298,6 +384,41 @@ export const VaultResourcesList = ({
       ) : (
         <></>
       )}
+      {selectedDataSource && selectedDataSource.dataSource.connector && (
+        <>
+          <ConnectorPermissionsModal
+            owner={owner}
+            connector={selectedDataSource.dataSource.connector}
+            dataSource={selectedDataSource.dataSource}
+            isOpen={showConnectorModal && !!selectedDataSource}
+            onClose={() => {
+              setShowConnectorModal(false);
+            }}
+            setShowEditionModal={setShowEditionModal}
+            handleUpdatePermissions={handleUpdatePermissions}
+            plan={plan}
+            readOnly={readOnly}
```

Comment:
I _think_ we don't want that right? Only admins should be there now?

Reply:
I'm a tiny bit confused with "who can see what" I have to admit.. 😅

Reply:
I understand, it's clearly not easy. 
=> So these modal are the modal to set up the connections right, which is only doable by admins from the System vault, right? 
And since the system vault is only accessible by admin, no one with another role should be able to land here. 


Before vaults it had a value to be able to show the connection modal in read mode because that was the only way to allow builders knowing which data from connections was shared with Dust, now they can do that from the Global vault. Does that make sense?

Reply:
That means we need to make sure that we don't prevent the Builders not migrated to Vault to still access the tree view in read mode, but when in Vault you must be an admin to access these modals.

In any case, once in front-edge I think it's worth checking this with Ed to make sure this is the expected XP both for people in Vault, and those you are not yet!

Reply:
Makes sense 🤓

Reply:
The current state of this PR seems to be on par with prod 😃